src/main/webapp/pages/projects/project_new.html

```diff
@@ -213,6 +220,32 @@ <h2 class="panel-title" th:text="#{projects.create.heading}">Create a New Projec
           }
         });
 
+        /*
+            Disables/enables the lockSamples checkbox
+            depending on if using the samples from cart
+        */
+        $("#useCartSamples").on("change", function(e) {
+          var selected = $(e.target).prop("checked");
+          if (!selected) {
+            $("#lockSamples").attr("disabled", true);
+          } else {
+            $("#lockSamples").removeAttr("disabled");
+          }
+        });
+
+        /*
+            If lockSamples checkbox is checked then we set
+            it's value to false which will set
+            the sample owner to false
+        */
+        $("#lockSamples").on("change", function(e) {
+          var selected = $(e.target).prop("checked");
+          if (selected) {
+            $("#lockSamples").val("false");
```

Comment:
You can test to be sure, but with the `@RequestParam(name = "lockSamples", required = false, defaultValue = "true") boolean owner` on the controller i'm pretty sure you don't need this at all.  If the checkbox is checked, `owner` will always be set to false, if it's unchecked it'll by default be true.

Reply:
Fixed in 94d58a64cfc89091be01471a00966a4ad200ad8e